jest.config.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+export default {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/r4/b1879v110c79t2wv163m95300000gp/T/jest_dy",
```

Comment:
As a user, I hate folder names like that, when inevitably they are left behind after the causing software might be long gone.

Can we use a human-readible name here? `aurelia-components`, say?

Reply:
I dont know if this is supposed to be human-readable, because this file was auto-generated by jest. And I actually don't care because its commented out.

Reply:
If it is auto-generated, can it be left out and added to `.gitignore` instead?

Reply:
I guess this path is just an example created by ‘jest init‘ on the first time.

Reply:
Fair enough.